zipkin-collector/activemq/src/main/java/zipkin2/collector/activemq/LazyInit.java

```diff
@@ -58,6 +58,7 @@ void close() {
     if (maybe != null) result.close();
   }
 
+  @SuppressWarnings("EmptyCatch")
```

Comment:
This is fine too though I think `// Ignore` in the catch block is fine too and probably what this check is intending

Reply:
yeah generally I am just trying to get the hugely annoying errorprone messages to go away one way or another. if an empty comment or // ignore works that works for me.